### PR TITLE
Use interface from config.xml when looking up ip address for iptables…

### PR DESCRIPTION
**Pull request by kangolo**
I was running into issues with my docker0 ip address being used for the iptables rule. This patch uses the first ethernet/wifi interface from config.xml instead of just the first interface returned by ifconfig.

### CODE REVIEW

**Comment by nohatssir** [comment]
It would be great if you could take this one step further and make it possible to select which interface to use from the web interface.

**Comment by kangolo** [comment]
Sure, I'll take a look - might take me a few days.

**Comment by kangolo** [comment]
Done, let me know if anything looks off or if the new field should be renamed.

**Comment by kangolo** [comment]
OK, I think I got everything. I left the get_ip function as-is as we'd still have to do the ifconfig call for the netmask and for troubleshooting down the line it makes sense for them to both use the same mechanism. I made two other minor changes:

I changed the capitalization on the label to match the rest of the page - "Gateway interface" rather than "Gateway Interface".

I moved the gateway_interface setting to be above the default_gateway setting so it's clear it's still required even if default_gateway is off/false.

File: var/www/openmediavault/js/omv/module/admin/service/openvpn/Settings.js [hunk 1]
```diff
@@ -17,10 +17,16 @@
...
+// require("js/omv/Rpc.js")
+// require("js/omv/data/Store.js")
+// require("js/omv/data/Model.js")
+// require("js/omv/data/proxy/Rpc.js")
 
 Ext.define('OMV.module.admin.service.openvpn.Settings', {
     extend: 'OMV.workspace.form.Panel',
-
+    requires: [
+            "OMV.Rpc"
```
Comment (nohatssir) [inline var/www/openmediavault/js/omv/module/admin/service/openvpn/Settings.js]: This line should have single quotation marks.

**Comment by nohatssir** [comment]
Looks fine now except for one line. When you've fixed that last issue commented on I'm fine with merging it. :)

**Comment by kangolo** [comment]
Done, thanks!

**Comment by nohatssir** [comment]
Thanks for the improvement! 👍

**Pull request merged**